DEPS

```diff
@@ -315,7 +315,7 @@ deps = {
   'src/third_party/googletest':
    Var('chromium_git') + '/external/github.com/google/googletest' + '@' + '7f036c5563af7d0329f20e8bb42effb04629f0c0',
 
-  'src/third_party/boringssl':
+  'src/flutter/third_party/boringssl':
```

Comment:
Oh yeah, `boringssl` is a little weird. `src/third_party/boringssl/src` down on 324 needs to be shifted at the same time.